release/smoke-tests/docker-compose.yml

```diff
@@ -7,7 +7,7 @@ services:
     working_dir: /usr/share/data-prepper/
     volumes:
       - ./data-prepper/config/data-prepper-config.yaml:/usr/share/data-prepper/config/data-prepper-config.yaml
-      - ./data-prepper/config/pipelines.yaml:/usr/share/data-prepper/pipelines/pipelines.yaml
+      - ./data-prepper/config/pipelines.yaml:/usr/share/data-prepper/pipelines.yaml
```

Comment:
The `pipelines/` dir was added to accommodate changes in 2.0. Is it causing you trouble to test a previous version? 

If that's the case, can we add a line without `pipelines/` dir instead? That might make it work for both cases.

Reply:
Should the current main branch read from `usr/share/data-prepper/pipelines/pipelines.yaml` or is that a planned change?

Reply:
Current main branch should do that.

Reply:
Thanks! I found the issue. When running the smoke test, it used an image I built previously.